package.json

```diff
@@ -53,23 +53,23 @@
     "@babel/preset-react": "^7.8.3",
     "@babel/register": "^7.8.6",
     "@rollup/plugin-replace": "^2.3.1",
-    "@testing-library/dom": "^6.8.1",
-    "@testing-library/react": "^9.3.2",
+    "@testing-library/dom": "^6.15.0",
```

Comment:
Please revert these so that the lockfile doesn't change. I don't think we need to change dependencies in this PR?

Reply:
Just did and pushed your suggested changes. Didn't merge yours.

Reply:
there is other dependencies that have changed. I thought I fixed these last night to be honest.